charts/csiplugin/templates/csi-attacher-opensdsplugin.yaml

```diff
@@ -39,7 +39,7 @@ spec:
       serviceAccount: csi-attacher
       containers:
         - name: csi-attacher
-          image: quay.io/k8scsi/csi-attacher:v1.0.1
+          image: quay.io/k8scsi/csi-attacher:v0.4.0
```

Comment:
Please use [v1.0.1](https://github.com/kubernetes-csi/external-attacher/releases/tag/v1.0.1) because it supports stable v1.0.0 csi spec and is supported by k8s v1.13.